docs/get-started/docker.md

```diff
@@ -6,6 +6,7 @@
 
 ```shell
 git clone git@github.com:pflooky/data-caterer-example.git
+./gradlew clean build
```

Comment:
yeah :). Just changed it to use the `./run.sh` instead to make it simpler. Thanks